gravitee-plugin-core/src/main/java/io/gravitee/plugin/core/internal/PluginRegistryImpl.java

```diff
@@ -221,6 +238,40 @@ private void loadPlugin(File registryDir, Path pluginArchivePath) {
         }
     }
 
+    /**
+     * Check if plugin is enabled.
+     * First, check in {@link PluginRegistryImpl#PLUGIN_TYPE_PROPERTY_ALIASES} plugin type has an alias for properties.
+     * Alias matches the implementation of {@link AbstractPluginHandler#type()}: plugin.type will be "service" but pluginHandler.type will be "services".
+     * If property is not contained based on alias, do a regular search of the property.
+     * @param pluginManifest
+     * @return true if plugin is enabled
+     */
+    private boolean isEnabled(PluginManifest pluginManifest) {
+        boolean enabled;
+        if (
+            PLUGIN_TYPE_PROPERTY_ALIASES.containsKey(pluginManifest.type()) &&
+            environment.containsProperty(
+                String.format(PROPERTY_STRING_FORMAT, PLUGIN_TYPE_PROPERTY_ALIASES.get(pluginManifest.type()), pluginManifest.id())
```

Comment:
We could certainly put that in a variable to make it easier to read (it seems used twice, in the if and inside de if).